### DISCUSSION TITLE
Rename Hudson to Jenkins

### DISCUSSION THREAD

**Discussion by troizet** [apache/netbeans#4087] [Ideas]
Hello! According to wikipedia (https://en.wikipedia.org/wiki/Hudson_(software)), Hudson has been declared obsolete since 2017. Perhaps it would be better if everywhere in ui renamed Hudson to Jenkins?

**Comment by mbien** [comment]
hi @troizet,

yes its probably time to update this ;)

Since Hudson is officially unmaintained and even eclipse (the last maintainer) switched to Jenkins, I believe simple renaming of all occurrences is indeed the right choice.

If you would like to work on this, please post on the dev mailing list first, in case there needs to be more discussion.

**Reply by mbien** [reply]
we got a contribution here: #6797, which does the renaming

**Comment by troizet** [comment]
The renaming Hudson to Jenkins has been done in #6797, so I am closing this discussion. Thanks to all involved!
